endpoint.go

```diff
@@ -890,6 +890,14 @@ func CreateOptionPortMapping(portBindings []types.PortBinding) EndpointOption {
 	}
 }
 
+// CreateOptionDNS function returns an option setter for dns entry option to
+// be passed to container Create method.
+func CreateOptionDNS(dns []string) EndpointOption {
+	return func(ep *endpoint) {
+		ep.generic[netlabel.DNSServers] = dns
+	}
+}
+
```

Comment:
External nameservers are not a property of the endpoints. Since the windows DNS server will only try to resolve the container names and not doing the proxying (ie: forwarding queries to external servers) why is this necessary ?